fighthealthinsurance/common_view_logic.py

```diff
@@ -1547,6 +1552,16 @@ async def generate_appeals(cls, parameters) -> AsyncIterator[str]:
             .aget()
         )
 
+        #TODO Collapse asave to the one after all the ml and pubsub work OR remove if get info update from final form
+        # Update the denial object with the received parameter if it differs
+        if denial.professional_to_finish != professional_to_finish:
+            logger.info(f"Updating denial {denial.denial_id} professional_to_finish from {denial.professional_to_finish} to {professional_to_finish}")
+            denial.professional_to_finish = professional_to_finish
+            await denial.asave(update_fields=['professional_to_finish']) 
```

Comment:
This could be part of why were seeing the locking issue, there's another PR earlier where I moved to aupdate for a lot of these individual changes. It's not critical but makes it easier in dev.